### DISCUSSION TITLE
How can I lower the concurrent tasks in specific op of the graph?

### DISCUSSION THREAD

**Discussion by EtienneT** [dagster-io/dagster#14851] [Q&A] [answered]
I have a dynamic @graph where somewhere I download a zip file.  If I download too many of those files concurrently, the server returns an error and I just get a bunch of invalid zip files.

I want to let the concurrency go to the default # of threads for most ops, but for those download ops, I want to limit it.  Where should I change that?

Thanks!

**Comment by hai-benwilson** [comment, marked as answer]
HI @EtienneT ,

You may want to check out [Dagster Concurrency](https://docs.dagster.io/guides/limiting-concurrency-in-data-pipelines). 
In your job config yaml, as example, you could set:
```yaml
execution:
  config:
    multiprocess:
      max_concurrent: 5
```
in combination with [DynamicGraphs](https://docs.dagster.io/concepts/ops-jobs-graphs/dynamic-graphs#dynamic-graphs). This would tell dagster, to only spin up 5 processes in parallel for the op running your processing code
